api/tests/integration/edge_api/identities/test_edge_identity_featurestates_viewset.py

```diff
@@ -1033,3 +1044,135 @@ def _create_segment_override(
         content_type="application/json",
     )
     assert create_segment_override_response.status_code == status.HTTP_201_CREATED
+
+
+def test_edge_identity_clone_flag_states_from(
+    admin_client: APIClient,
+    app_settings_for_dynamodb: None,
+    dynamo_enabled_environment: int,
+    dynamo_enabled_project: int,
+    environment_api_key: str,
+    flagsmith_identities_table: Table,
+    features_for_identity_clone_flag_states_from: typing.Callable,
+    featurestates_urls: dict[str, Template],
+) -> None:
+    def create_identity(identifier: str) -> EdgeIdentity:
+        identity_model = IdentityModel(
+            identifier=identifier,
+            environment_api_key=environment_api_key,
+            identity_features=IdentityFeaturesList(),
+            identity_uuid=uuid.uuid4(),
+        )
+        return EdgeIdentity(engine_identity_model=identity_model)
+
+    # GIVEN
```

Comment:
```suggestion
    # Given
```

Reply:
I removed some of the inline comments and applied Zack suggestions.